packages/wrangler/src/d1/list.tsx

```diff
@@ -12,18 +12,26 @@ import type {
 import type { Database } from "./types";
 
 export function Options(d1ListYargs: CommonYargsArgv) {
-	return d1ListYargs.epilogue(d1BetaWarning);
+	return d1ListYargs
+		.option("json", {
+			describe: "return output as clean JSON",
+			type: "boolean",
+		})
+		.epilogue(d1BetaWarning);
 }
 
-export async function Handler(
-	_: StrictYargsOptionsToInterface<typeof Options>
-): Promise<void> {
+export async function Handler({
+	json,
+}: StrictYargsOptionsToInterface<typeof Options>): Promise<void> {
 	const accountId = await requireAuth({});
-	logger.log(d1BetaWarning);
-
 	const dbs: Array<Database> = await listDatabases(accountId);
 
-	render(<Table data={dbs}></Table>);
+	if (json) {
+		logger.log(JSON.stringify(dbs, null, 2));
+	} else {
+		logger.log(d1BetaWarning);
```

Comment:
Should this beta warning be moved above the if statement?

Reply:
My idea was to not print the warning since it might pollute stdout

Reply:
Ok I see the use case you may be referring to in regards of maybe piping the stdout to say a `.json` file or something and this messaging would break that.